apps/src/templates/CollapserIcon.story.jsx

```diff
@@ -11,41 +11,41 @@ const styles = {
 export default storybook => {
   return storybook.storiesOf('CollapserIcon', module).addStoryTable([
     {
-      name: 'CollapserIcon in normal-mode, extended',
+      name: 'CollapserIcon default, expanded',
       story: () => (
         <div style={styles.background}>
-          <CollapserIcon onClick={() => {}} collapsed={false} />
+          <CollapserIcon onClick={() => {}} isCollapsed={false} />
         </div>
       )
     },
     {
-      name: 'CollapserIcon in normal-mode, collapsed',
+      name: 'CollapserIcon default, collapsed',
       story: () => (
         <div style={styles.background}>
-          <CollapserIcon onClick={() => {}} collapsed={true} />
+          <CollapserIcon onClick={() => {}} isCollapsed={true} />
         </div>
       )
     },
     {
-      name: 'CollapserIcon in teacher-mode, extended',
+      name: 'CollapserIcon different icon, expanded',
       story: () => (
         <div style={styles.background}>
           <CollapserIcon
             onClick={() => {}}
-            collapsed={false}
-            teacherOnly={true}
+            isCollapsed={false}
+            expandedIconClass="fa-caret-down"
           />
         </div>
       )
     },
     {
-      name: 'CollapserIcon in teacher-mode, collapsed',
+      name: 'CollapserIcon custom style, expanded',
       story: () => (
         <div style={styles.background}>
           <CollapserIcon
             onClick={() => {}}
-            collapsed={true}
-            teacherOnly={true}
+            isCollapsed={false}
+            style={{color: 'red'}}
```

Comment:
What is the story behind the red choice?

Reply:
Oh it was just an arbitrary choice, I just wanted it to show that the collapser icon can be custom styled and red seemed like a clear way to display it as custom.

Reply:
i dig